addons/packages/metrics-server/0.5.1/bundle/config/overlays/update_overlay.yaml

```diff
@@ -0,0 +1,31 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
```

Comment:
And if this is sort of "universal to all packages"  type of file, it would be nice if it was either:
1.) copy in 1 place outside each package, rather than duplicating and have it applied as an additional templating step.
or 2.) stored in a directory within `/overlays` as like `/overlays/system` or `/overlays/tkg-system`.  

At some point I imagine there is bound to be a template ordering bug, so knowing the "system" templates are applied as a separate step could be helpful.